### PR TITLE
Added my photo.

### PR DESCRIPTION
**Pull request by dhruvramani**

### CODE REVIEW

**Comment by roonyh** [comment]
Thanks much! Almost ready! please fix the indentation so that it looks nicer. And use only space characters please don't use 'tab' characters.

File: index.html [hunk 1]
```diff
@@ -1028,6 +1028,24 @@ <h1 class="person__name">Phan Duy Duc</h1>
 			</section>
 		    </div>
 		</div>
+	<div class="sympl onethird">
```
Comment (roonyh) [inline index.html]: wrong indentation here.

File: index.html [hunk 2]
```diff
@@ -1028,6 +1028,24 @@ <h1 class="person__name">Phan Duy Duc</h1>
...
+                    <div class="box-team wow fadeIn" data-wow-delay="0.1s">
+                        <section class="person person--student">
+                            <div class="person__img">
+                                <div class="person__img__info">
+                                    <div class="person__img__info__cont">
+                                        <a href="https://www.facebook.com/dhruv.ramani" target="_blank"><i class="icon-facebook"></i></a>
+                                        <a href="https://github.com/dhruvramani" target="_blank"><i class="icon-github"></i></a>
+                                       <a href="mailto:dhruvramani98@gmail.com" target="_blank"><i class="icon-mail"></i></a>
```
Comment (roonyh) [inline index.html]: and here

**Comment by dhruvramani** [comment]
Okay.

**Comment by dhruvramani** [comment]
Can someone please merge this? I am waiting for a long time now..

**Comment by roonyh** [comment]
hey i'll merge this asap. btw your task has been closed for a while. you
are free to claim another task.

On Thu, Dec 25, 2014 at 9:22 PM, Dhruv Ramani notifications@github.com
wrote:

> Can someone please merge this? I am waiting for a long time now..
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/fossasia/fossasia.github.io/pull/134#issuecomment-68106769
> .

## 

Regards,
Aruna Herath

**Pull request merged**